web_src/js/utils/color.js

```diff
@@ -33,3 +31,13 @@ export const chartJsColors = resolveColors({
   additions: '--color-green',
   deletions: '--color-red',
 });
+
+function hex(x) {
+  const hexDigits = ['0', '1', '2', '3', '4', '5', '6', '7', '8', '9', 'a', 'b', 'c', 'd', 'e', 'f'];
+  return Number.isNaN(x) ? '00' : hexDigits[(x - x % 16) / 16] + hexDigits[x % 16];
+}
+
+export function rgbToHex(rgb) {
+  rgb = rgb.match(/^rgba?\((\d+),\s*(\d+),\s*(\d+).*\)$/);
+  return `#${hex(rgb[1])}${hex(rgb[2])}${hex(rgb[3])}`;
```

Comment:
So we simply ignore the `alpha` when it exists?

Reply:
I did not modify that function, it was just moved as-is from the projects file.

Reply:
But who says that this function will only be used for projects?
It does not sound like only being intended for projects…

Reply:
Hmm let me check if we maybe can get rid of it..

Reply:
Done in https://github.com/go-gitea/gitea/pull/30237/commits/4ac99163d0e966960ab3465b1046daa5af922848.